src/server/handlers/index.ts

```diff
@@ -190,6 +192,31 @@ export default {
       ['success']
     ]
   ),
+  GetServicesByServiceType: wrapWithHistogram(
+    Services.get,
+    [
+      'servicesServiceType_get',
+      'Put servicesServiceType request',
+      ['success']
+    ]
+  ),
+  PutServicesByServiceType: wrapWithHistogram(
+    Services.put,
+    [
+      'servicesServiceType_put',
+      'Put servicesServiceType request',
+      ['success']
+    ]
+  ),
+  // todo: fix operation id
```

Comment:
nit: todo in here

Reply:
These todo's require me to go back and fix some of the dependency libraries which I considered low-priority vs getting this in. But I might as well not leave this tech debt and do that now since you're on early.

Reply:
No worries - leave them if you want, maybe just open a ticket and make a reference to it in case you don't get around ti it.